src/components/RateArticles/RateArtcles.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import StarRatingComponent from 'react-star-rating-component';
+import PropTypes from 'prop-types';
+
+class ArticleRating extends React.Component {
+  state = {};
+
+  onStarClick = nextValue => {
+    const { article, starClick } = this.props;
+    const slug = article.slug;
+    starClick(slug, { rate: nextValue }, rating =>
+      this.setState({
+        rating
+      })
+    );
+  };
+
+  render() {
+    const { article, currentUser } = this.props;
+    return !!currentUser?(
```

Comment:
Hello @Philipotieno ,
When you negate !!currentUser==currentUser, you need not to use the double negation